pom.xml

```diff
@@ -248,6 +219,103 @@
+                                <module.path>${jboss.modules.path}</module.path>
+                                <arquillian.xml>${maven.multiModuleProjectDirectory}/arquillian.xml</arquillian.xml>
+                            </systemPropertyVariables>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <!--
+            When it comes to non-XP EAP testing then only the MicroProfile 
+            specs belonging to the so called "observability layer" should be 
+            tested, hence this profile should be activated by providing the 
+            `mp.specs.scope` property with the "observability-layer" value
+        -->
+        <profile>
+            <id>observability-layer-modules</id>
+            <activation>
+                <property>
+                    <name>mp.specs.scope</name>
+                    <value>observability</value>
+                </property>
+            </activation>
+            <modules>
+                <module>tooling-cpu-load</module>
+                <module>tooling-docker</module>
+                <module>tooling-mp-jwt-auth-tool</module>
+                <module>tooling-server-configuration</module>
+                <module>microprofile-config</module>
+                <module>microprofile-health</module>
+                <module>microprofile-metrics</module>
+            </modules>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <version>${version.org.apache.maven.plugins.maven-surefire-plugin}</version>
+                        <configuration>
+                            <environmentVariables>
+                                <JBOSS_HOME>${jboss.home}</JBOSS_HOME>
+                            </environmentVariables>
+                            <systemPropertyVariables>
+                                <jboss.home>${jboss.home}</jboss.home>                                
+                                <jboss.configuration.file>standalone.xml</jboss.configuration.file>
+                                <container.base.dir.manual.mode>${container.base.dir.manual.mode}</container.base.dir.manual.mode>
+                                <module.path>${jboss.modules.path}</module.path>
+                                <arquillian.xml>${maven.multiModuleProjectDirectory}/arquillian.xml</arquillian.xml>
+                            </systemPropertyVariables>
+                            <!-- MP Health integration tests with MP Fault Tolerance 
+                            and MP Metrics must be excluded when testing just 
+                            the MP "observability layer" specs-->                            
```

Comment:
What is the reason for this? Is it because tests are integrating with other specs?

Reply:
Exactly, integration tests with MP FaultTolerance must be excluded, as this feature isn't available when running a server which would integrate just the observability layer specs.